### PR TITLE
13849 handle empty address for conversion filing

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#13849

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/store/getters/state-getters.ts [hunk 1]
```diff
@@ -742,10 +742,14 @@ export const hasRecDeliveryChanged = (state: StateIF): boolean => {
...
-    if (!op.mailingAddress.deliveryInstructions) op.mailingAddress.deliveryInstructions = ''
-    if (!op.mailingAddress.streetAddressAdditional) op.mailingAddress.streetAddressAdditional = ''
+    if (op.deliveryAddress) {
+      if (!op.deliveryAddress.deliveryInstructions) op.deliveryAddress.deliveryInstructions = ''
+      if (!op.deliveryAddress.streetAddressAdditional) op.deliveryAddress.streetAddressAdditional = ''
+    }
+    if (op.mailingAddress) {
+      if (!op.mailingAddress.deliveryInstructions) op.mailingAddress.deliveryInstructions = ''
+      if (!op.mailingAddress.streetAddressAdditional) op.mailingAddress.streetAddressAdditional = ''
+    }
```
Comment (severinbeauvais) [inline src/store/getters/state-getters.ts:752]: This looks good, but did you see this code and would it be a good place to init a blank address?

https://github.com/bcgov/business-edit-ui/blob/51716ef295600d0a7b7aa0fb9293dca0b9d595f2/src/views/Conversion.vue#L210

Comment (vysakh-menon-aot) [reply src/store/getters/state-getters.ts:752]: that make sense. will update

Comment (severinbeauvais) [reply src/store/getters/state-getters.ts:752]: I'm not sure which is better (more concise for the current issue, clearer to understand)... leaving it with you. 😊

Comment (vysakh-menon-aot) [reply src/store/getters/state-getters.ts:752]: its failing without this if conditions. so not changing it

Comment (severinbeauvais) [reply src/store/getters/state-getters.ts:752]: OK

**Review by severinbeauvais — APPROVED** [review]

**Review by argush3 — APPROVED** [review]

**Pull request merged**